x/emissions/types/keys.go

```diff
@@ -51,5 +51,14 @@ var (
 
 	// BallotMaturityBlocks is amount of blocks needed for ballot to mature
 	// by default is set to 100
-	BallotMaturityBlocks = 100
+	BallotMaturityBlocks = 100 // approximately 9-10 minutes
+	// PendingBallotsBufferBlocks is a buffer number of blocks
+	//(in addition to BallotMaturityBlocks)
+	// that we use only for pending ballots before deleting them
+	PendingBallotsBufferBlocks = SecondsToBlocks(60 * 60 * 24 * 10) // 10 days
```

Comment:
Decided to use 10 days instead of 2 weeks, imo 10 days seems to be a good enough time frame